Asterisk.2013/Asterisk.NET/Manager/ManagerReader.cs

```diff
@@ -223,7 +235,7 @@ internal void Run()
 							if (lastPacketTime.AddMilliseconds(mrConnector.PingInterval) < DateTime.Now
 								&& mrConnector.PingInterval > 0
 								&& mrSocket != null
-								&& !wait4identiier

```

Comment:
I think about it.
But we have this line:
`if (wait4identiier && line.StartsWith("Asterisk Call Manager"))`
I decide left wait4identiier because i don't know how it will affect logic above.
I think if we get 'welcome line', when we not wait it,this is a not correct situation and we need crush.